000-build-programs/lfx-2024-term3.json

```diff
@@ -0,0 +1,769 @@
+{
+  "term": "lfx-2024-term3",
+  "startDate": "2024-09-09",
+  "endDate": "2024-11-29",
```

Comment:
I think this is the date format we're using -- i'm not sure i got it correct for term 2...